bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/Parameters.scala

```diff
@@ -108,84 +108,79 @@ sealed trait CalculationMethod {
   require(isSpectroscopy != isImaging)
 }
 
+// Calculations that return signal to noise out of an amount of exposures
```

Comment:
Do you not also need the exposure time to do this?

Reply:
yes